pypowsybl/loadflow.py

```diff
@@ -187,7 +187,7 @@ def __repr__(self) -> str:
                f")"
 
 
-def run_ac(network: _Network, parameters: Parameters = None, provider : str = 'OpenLoadFlow') -> _List[ComponentResult]:
+def run_ac(network: _Network, parameters: Parameters = None, provider: str = '') -> _List[ComponentResult]:
```

Comment:
We need to update the docstring: OpenLoadFlow is not anymore the default, depending on the new config (same for run_dc).